### PR TITLE
Package WsgiDAV

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Retry of #9676

### CODE REVIEW

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
> @conda-forge-admin, please rerender

The conda forge admin commands are only required once the package is integrated in conda-forge, for the submission to staged-recipes they are not required.

File: recipes/wsgidav/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 415ac221806a266a3661f6f2f91ecc8dbcbe3653cdb1508abc0c397ea0cf35f3
+  patches:
+  - patches/0001-remove-setup_requires.patch
+  - patches/0002-use-cx_freeze-on-windows.patch  # [win]
+
+build:
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (jan-janssen) [inline recipes/wsgidav/meta.yaml]: The `pip install .` command looks alright, but somehow the package builds a wheel rather than installing from source.

**Comment by h-vetinari** [comment]
> @jan-janssen: The conda forge admin commands are only required once the package is integrated in conda-forge, for the submission to staged-recipes they are not required.

Yeah, I realised when nothing happened. Force of habit. ;-)

> @jan-janssen: The `pip install .` command looks alright, but somehow the package builds a wheel rather than installing from source.

Yeah, the `setup.py` is quite involved. I also need to fix (figure out) how to deal with the jinja2-vendoring on windows. Any help is appreciated.

**Pull request closed without merging**